packages/styleguide/src/components/Figure/SwitchImage.js

```diff
@@ -11,6 +11,7 @@ const SwitchImage = ({ src, srcSet, dark, alt, ...props }) => {
         src={src}
         srcSet={srcSet}
         alt={alt}
+        style={{ ...props.style, maxWidth }}
```

Comment:
If it wasn't applied previously (as maxWidth is no valid attribute), are we sure that everything is still behaving the same way as before?

Reply:
If only we had something like tests that would catch this 😏 But yeah, I wondered too. I checked the current prod/love fronts by taking full-page screenshots.

I think it doesn't really matter because maxWidth is always absurdly high (> 1000). I'm not sure why it's used at all but we can look at that later.